src/components/BrokenFirmwareDialog.tsx

```diff
@@ -68,14 +70,14 @@ const BrokenFirmwareDialog = ({
               <Text w="100%">
                 <Link
                   color="brand.600"
-                  href="https://microbit.org/get-started/user-guide/firmware/"
```

Comment:
Spotted this link was duplicated. The text is the generic troubleshooting copy for this 2nd link so should be the same support article.